UI/window-basic-settings.cpp

```diff
@@ -4844,9 +4862,11 @@ void OBSBasicSettings::SimpleStreamingEncoderChanged()
 		defaultPreset = "balanced";
 		preset = curQSVPreset;
 
-	} else if (encoder == SIMPLE_ENCODER_NVENC) {
-		obs_properties_t *props =
-			obs_get_encoder_properties("ffmpeg_nvenc");
+	} else if (encoder == SIMPLE_ENCODER_NVENC ||
+		   encoder == SIMPLE_ENCODER_NVENC_HEVC) {
+		obs_properties_t *props = obs_get_encoder_properties(
+			encoder == SIMPLE_ENCODER_NVENC ? "ffmpeg_nvenc"
+							: "ffmpeg_hevc_nvenc");
```

Comment:
The HEVC parts of this should be behind an ifdef.

Reply:
@RytoEX 
This was done in order to have a bit simply code.
`#define SIMPLE_ENCODER_NVENC_HEVC` does not generate the code, so is does not violate any HEVC restrictions. 

#ifedf/endif pairs are already too many for effective code maintenance. Minus two of them (and two more for NVENC and two for AMD later) looks like a good idea.

So, should I move this define under #ifdef? I agree with any decision.
Just described the source of the solution to such code.

Reply:
I would _largely_ prefer that _any_ HEVC code is behind the ENABLE_HEVC ifdef, regardless of how much more difficult it is.

Reply:
The else if block should be copy/pasted anyway because I think it's likely we'll want to process the property list differently when we understand the settings better.

Reply:
Done.